float32 velocity_scale_factor
float32 acceleration_scale_factor
---
bool success